common/file_helpers/file.go

```diff
@@ -11,8 +11,8 @@ import (
 
 // FileExists checks if the file exist or not
 func FileExists(path string) bool {
-	_, err := os.Stat(path)
-	if err == nil {
+	file, err := os.Stat(path)
+	if err == nil && !file.IsDir() {
```

Comment:
This looks like it might raise a bug when helpers.ConfigDir() tries to use this method to test for the new config dir existing.

Reply:
Yeah you're right. Thanks for the catch. I am thinking we could change rename the original method to PathExist then create a new method called FileExists using my new changes. This way there is clarity on what each method is expected to do. Where PathExits verify if the path is either a file or dir and FileExists check verify if the file exists. Thoughts?

Reply:
@jorge-ibm I agree with the reasoning. I think for the sake of this bug I think it would better to keep it specific to the CLI code. I will close this PR  and make the adjustments there.